libbeat/cfgfile/cfgfile.go

```diff
@@ -76,6 +111,7 @@ type ConditionalOverride struct {
 // ChangeDefaultCfgfileFlag replaces the value and default value for the `-c`
 // flag so that it reflects the beat name.
 func ChangeDefaultCfgfileFlag(beatName string) error {
+	Initialize()
```

Comment:
[Suggestion]
Update the comment of this function adding that it also calls `Initialize` or describing that it handles the flags.

Reply:
added.